src/entities/trainer/ui/trainers-card/trainer-card.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames/bind';
+import { Trainer } from '../../types';
+import { Image } from '@/shared/ui/image';
+
+import styles from './trainer-card.module.scss';
+
+const cx = classNames.bind(styles);
+
+type TrainerProps = Trainer;
```

Comment:
redundant type

Reply:
Is there any way to rewrite it, or you just suggest using Trainer as a type for props?
Im not entirely sure if it's good, because it's just coincidence that the type of etity is completely the same as component props type.

Reply:
I don't see any downsides using this type directly as props type. You explicitly will tell that the component accept trainer's type of data